python3/frodokem.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, variant = "FrodoKEM-640-AES"):
             self.gen = self.genSHAKE128
         else:
             assert "Unknown variant"
+        print("WARNING: This Python3 implementation of FrodoKEM is not designed to be fast or secure, and may leak secret information via timing or other side channels; it should not be used in production environments.")
```

Comment:
perhaps do this via [`warnings.warn`](https://docs.python.org/3/library/warnings.html)?

Reply:
Done.